packages/ui/src/SlotTypeIcon/SlotTypeIcon.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import { Ice, OffIce } from "@eisbuk/svg";
+
+import { SlotType } from "@eisbuk/shared";
+import { useTranslation, SlotTypeLabel } from "@eisbuk/translations";
+
+const iconLookup = {
+  [SlotType.Ice]: Ice,
+  [SlotType.OffIce]: OffIce,
+};
+
+const SlotTypeIcon: React.FC<{ type: SlotType; className?: string }> = ({
+  type,
+  className = "",
+}) => {
+  const { t } = useTranslation();
+
+  const Icon = iconLookup[type];
+
+  return (
+    <div className={[...baseClasses, className].join(" ")}>
+      <span key="icon" className="w-5 h-5 mr-2">
+        <Icon />
+      </span>
+      <span key="type">{t(SlotTypeLabel[type])}</span>
+    </div>
+  );
+};
+
+const baseClasses = [
+  "px-4",
+  "py-3",
+  "font-sm",
+  "uppercase",
+  "text-base",
+  "text-bold",
+  "text-gray-500",
+  "flex",
+  "flex-row",
+  "items-center",
+  "justify-center",
+  "md:flex-row",
+  "md:px-4",
+  "md:py-3",
```

Comment:
Default flex is `flex-row` so no need to specify it. If you have `px-4` and `md:px-4`.